### PR TITLE
Fix readme to match cloud 2 standards

### PR DESCRIPTION
**Pull request by petedejoy**
This PR edits the language in our ReadMe to reflect how CLI versioning should work with new Cloud.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,31 +31,30 @@ Use "astro [command] --help" for more information about a command.
...
+To install a previous version of Astronomer, you can add the tag at the end of your `curl` command via the following syntax:
+
 ```
 curl -sSL https://install.astronomer.io | sudo bash -s -- [TAGNAME]
 ```
 
 ie:
 ```
-curl -sSL https://install.astronomer.io | sudo bash -s -- v0.3.1
+curl -sSL https://install.astronomer.io | sudo bash -s -- v0.7.5
```
Comment (andriisoldatenko) [inline README.md]: what do you think about 0.10.3?

Comment (schnie) [reply README.md]: Actually lets just drop the version here so it installs the latest stable by default. And we should change the url to be the new prod url for this.

Comment (petedejoy) [reply README.md]: So this is a section specifically for "Previous Versions". It comes after the "Latest Version" section which installs the latest stable by default.

What's the new prod URL?

**Comment by petedejoy** [comment]
Can we get this merged @andriisoldatenko @schnie? Know you're juggling the release right now but pretty confident it's an improvement on what we're showing currently.

File: README.md (hunk 1 above)
Comment (schnie) [reply README.md]: That's still it. There's always one launched with the platform that installs the platform version, but this on is separate and hosted on its own. It'll always do the latest published.

**Pull request merged**